response_operations_ui/templates/partials/mock-create-message-link.html

```diff
@@ -0,0 +1,12 @@
+<div class="grid">
```

Comment:
Is this necessary?  This HTML could just be in the reporting unit template with a comment saying 'remove me once X happens'.  This might get forgotten about when that partial is removed and then we'll have an unused file messing up the place